_grades/A9.md

```diff
@@ -0,0 +1,95 @@
+
+When presented with a number of solutions, you can quickly and confidently select the best option to allow your team to progress
+
+You can predict market trends and steer your team through architectural changes.
+
+You frequently attend technical conferences, absorb and share key messages across your teams
+
+Project software architects come to you for input
+
+## Soft Skills (T)
+
+You appreciate, and take into account, other teams’ perspective on technical projects (PMO, Operations etc). You have a good working relationship with these teams and they come to you for advice.
+
+You can gauge the wellbeing of your team and constantly work towards improving it
+
+You are a confident public speaker in your chosen domain, you can clearly articulate your thoughts
+
+## Getting Code Live (Str)
+
+You are able to provide advice on estimation in multiple environments
+
+You can ask the right questions to enable a team to generate confidence in their deployment process
+
+You have the right contacts in other teams to get help troubleshooting deployment issues on multiple platforms
+
+## Impact (Wis)
+
+### Solution Ownership
+
+You have been responsible for one or more large project releases
+
+You can identify and delegate to engineering leads across multiple teams
+
+Your recognisable ways of working can be seen across the teams in your projects
+
+You are not afraid to identify and correct solution deficiencies, even if you are far down a project timeline
+
+### Technical Assessment and Adoption
+
+You are able to evaluate and make decisions on technology propositions
+
+You keep up to date with industry direction, and contribute your opinions to the industry
+
+You delegate, and ensure progress on, team development towards your recommended technical directions
+
+### Business Awareness
+
+Your opinion is sought by the customer on solutions to improve business value
+
+You are aware of your business domain knowledge limitations and able to clearly define the lines / decision matrices within a team
+
+### Developing Our Team
+
+You assign reviewers to benefit the team, and ensure that the team has attainable targets to show progress throughout the year
+
+You encourage team initiatives and side-projects, and work to enable funding and time for these projects
+
+You work hard to promote the team as a great place to work, and to make sure that this promoted image is true
+
+You look for new pools of talent and grow the talent within the team
+
+### Winning New Work
+
+You are involved in the review and signoff of bids
+
+You create packages for the sales team to take out to clients
+
+You are the go-to person for client presentations, and you are able to delegate and hand over to team members that you know will be able to fill in for you
+
+## Team Leadership (Fel)
+
+You are a true Agile Coach
```

Comment:
The previous grade says about leading the scrum of scrums, so I would assume this is being an agile coach cutting across all of OSCE? Perhaps the wording should be updates a little if people agree...

Reply:
@sasaunde What are your views on the above?

Reply:
Something along these lines https://www.toptal.com/project-managers/agile/what-is-an-agile-coach